components/CityAutoComplete.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+	<div class="w-full" @focusout="handleFocusOut">
+		<AutoComplete
+			ref="autocompleteRef"
+			v-model="model"
+			:suggestions="citySuggestions"
+			:loader="undefined"
+			class="relative w-full rounded-none"
+			input-class="w-full md:rounded-none md:rounded-r-md rounded-md border-none focus:cursor-text cursor-pointer dark:!bg-dark-800"
+			aria-label="City name"
+			:delay="0"
+			@keyup.enter.stop="(event: KeyboardEvent) => emit('confirm', event)"
+			@keyup.esc="emit('cancel')"
+		>
+			<template #option="slotProps">
+				<span class="flex items-center gap-1">
+					<span>
+						{{slotProps.option}}
+					</span>
+					<span v-if="countryCode === 'US'" class="text-bluegray-400">
```

Comment:
Let's use this design here (comma and slightly smaller text, see location at https://globalping.io/): 
<img width="232" height="43" alt="image" src="https://github.com/user-attachments/assets/dc242ff5-fcc4-42c0-9d75-5f704b500502" />